.github/workflows/deploy.yml

```diff
@@ -3,8 +3,22 @@ on:
   workflow_call:
     inputs:
       environment:
+        description: 'The Github environment to load secrets from'
         type: string
         required: true
+      context:
```

Comment:
suggestion: a more descriptive variable, since it's actually very netlify specific? e.g. `netlify-context` (or `netlify_context`, whichever is the proper style).

same for the `alias` input below.